MagneticField/ParametrizedEngine/test/BuildFile.xml

```diff
@@ -0,0 +1,7 @@
+<bin file="testParabolicParametrizedMagneticField.cc">
+  <use name="alpaka"/>
+  <use name="HeterogeneousCore/AlpakaInterface"/>
+  <use name="eigen"/>
+  <use name="DataFormats/GeometryVector"/>
+  <use name="FWCore/Utilities"/>
+</bin>
```

Comment:
```suggestion
  <use name="FWCore/Utilities"/>
  <flags ALPAKA_BACKENDS="1"/>
</bin>
```
is missing (assuming the intention is to build one binary for each backend).

Reply:
Added: https://github.com/cms-sw/cmssw/pull/44360/commits/8d93f61cc051bd4b2a39d6559170dd00f6a2bcef